.github/workflows/int-and-connected-test-run.yml

```diff
@@ -276,11 +280,16 @@ jobs:
       - name: "Notify Jade Slack on nightly test run"
         if: ${{ github.event_name == 'schedule' && always() }}
         uses: broadinstitute/action-slack@v3.8.0
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
         with:
-          status: ${{ (needs.test_check.outputs.job-status == 'success' && needs.test_connected.outputs.job-status == 'success' && needs.deploy_test_integration.outputs.job-status == 'success') && 'success' || 'failure' }}
+          status: ${{ env.RUN_STATUS }}
           channel: "#jade-alerts"
           username: "Data Repo tests"
           text: "Nightly Unit, Connected and Integration tests"
+      - name: "Notify QA Slack on nightly test run"
+        if: ${{ github.event_name == 'schedule' && always() }}
```

Comment:
If a test fails and needs to be manually re-run, will it send a slack message in the QA channel? (Assuming we want it to do this)

Reply:
Yes it will! We do want to do this. We've verified the behavior with the existing Slack alert to #jade-alerts.